### PR TITLE
Improve style (line length, freeze constants).

### PR DESCRIPTION
**Pull request by ashercodeorg**

### CODE REVIEW

File: lib/cdo/pardot.rb [hunk 1]
```diff
@@ -44,82 +44,110 @@ class Pardot
...
-        id = node.xpath("id").text.to_i
+        pardot_id = node.xpath("id").text.to_i
         email = node.xpath("email").text
         results_in_response += 1
-        id_max = id
-        # update the Pardot ID for this contact in our database
-        PEGASUS_DB[:contact_rollups].where(email: email).update(pardot_id: id)
+        email_to_pardot_id[email] = pardot_id
+      end
+      id_max = email_to_pardot_id.values.max
```
Comment (ashercodeorg) [inline lib/cdo/pardot.rb]: This is an implementation change.

File: lib/cdo/pardot.rb [hunk 2]
```diff
@@ -44,82 +44,110 @@ class Pardot
...
+
+      # Update the Pardot ID in our DB for these contacts.
+      PEGASUS_DB.transaction do
+        email_to_pardot_id.each do |email, pardot_id|
+          PEGASUS_DB[:contact_rollups].
+            where(email: email).
+            update(pardot_id: pardot_id)
+        end
```
Comment (ashercodeorg) [inline lib/cdo/pardot.rb]: This is an implementation change.

**Comment by jeremydstone** [comment]
Per our conversation the thing you optimized consumes only a trivial amount of time and the optimization has never been tested or run, so let's not do that. The line ending changes are OK if you can execute and verify the code you are changing.

File: lib/cdo/pardot.rb (hunk 1 above)
Comment (ashercodeorg) [reply lib/cdo/pardot.rb]: Reverted.

File: lib/cdo/pardot.rb (hunk 2 above)
Comment (ashercodeorg) [reply lib/cdo/pardot.rb]: Reverted.

**Pull request merged**